mmv1/products/compute/RegionHealthSource.yaml

```
@@ -0,0 +1,162 @@
+      The name must be 1-63 characters long, and comply with RFC1035.
+      Specifically, the name must be 1-63 characters long and match the regular
+      expression `[a-z]([-a-z0-9]*[a-z0-9])?` which means the first
+      character must be a lowercase letter, and all following characters must
+      be a dash, lowercase letter, or digit, except the last character, which
+      cannot be a dash.
+
+properties:
+  - name: 'description'
+    type: String
+    description: |
+      An optional description of this resource. Provide this property when you
+      create the resource.
+  - name: 'sourceType'
+    type: Enum
+    description: |
+      Specifies the type of the `HealthSource`. The only allowed value
+      is `BACKEND_SERVICE`. Must be specified when the
+      `HealthSource` is created, and cannot be mutated.
+    enum_values:
+      - 'BACKEND_SERVICE'
+    required: true
+    immutable: true
+  - name: 'sources'
+    type: Array
+    item_type:
+      name: 'source'
+      type: ResourceRef
+      resource: 'RegionBackendService'
+      imports: 'selfLink'
+    description: |
+      URLs to the source resources. Must be size 1. Must be a
+      `BackendService` if the `sourceType` is `BACKEND_SERVICE`. The
+      `BackendService` must have load balancing scheme
+      `INTERNAL` or `INTERNAL_MANAGED` and must be regional
+      and in the same region as the `HealthSource` (cross-region
+      deployment for `INTERNAL_MANAGED` is not supported). The
+      `BackendService` may use only IGs, MIGs, or NEGs of type
+      `GCE_VM_IP` or `GCE_VM_IP_PORT`. The
+      `BackendService` may not use `haPolicy`. Can be
+      mutated.
+    min_size: 1
+    max_size: 1
+  - name: 'healthAggregationPolicy'
+    type: ResourceRef
+    resource: 'RegionHealthAggregationPolicy'
+    imports: 'selfLink'
+    description: |
+      URL to the `HealthAggregationPolicy` resource. Must be set. Must
+      be regional and in the same region as the `HealthSource`. Can be
+      mutated.
+  - name: 'id'
+    type: String
+    description: 'The unique identifier for the resource. This identifier is defined by the server.'
+    output: true
+  - name: 'creationTimestamp'
+    type: String
+    description: 'Creation timestamp in RFC3339 text format.'
+    output: true
+  - name: 'selfLinkWithId'
+    type: String
+    description: 'Server-defined URL with id for the resource.'
+    output: true
+  - name: 'fingerprint'
+    type: Fingerprint
+    description: |
+      Fingerprint of this resource. A hash of the contents stored in this object.
+      This field is used in optimistic locking. This field will be ignored when
+      inserting a `HealthSource`. An up-to-date fingerprint
+      must be provided in order to patch the HealthSource; Otherwise, the request
+      will fail with error `412 conditionNotMet`. To see the latest
+      fingerprint, make a `get()` request to retrieve the
+      HealthSource.
```

I think most of this isn't relevant to Terraform users, since they don't control what's sent to the API for this field.

```suggestion
      Fingerprint of this resource. A hash of the contents stored in this object.
      This field is used in optimistic locking.
```